src/app/modules/shared-components/components/message/message.component.html

```diff
@@ -0,0 +1 @@
+<p-messages [value]="msgs" [closable]="closable"></p-messages>
```

Comment:
It is generally admitted (for various reason) that files should end with line break.
BTW, have you installed the VScode module I recommended yesterday? Because the editorconfig file should not allow to do that.